index.html

```diff
@@ -232,6 +232,33 @@ <h3>A Special Time</h3>
               Subscribe
             </a>
           </p>
+          <h3>Please comment in the form below</h3>
+           <!--Comment Form-->
+          <div class="form">
+            <form action="/get_data" method="POST">
+                <label for="name">Name :</label>
+                <input type="text" id="name" name="user_name" maxlength="20" placeholder="Your full name" required>
+                <label for="email">Email :</label>
+                <input type="email" id="email" name="email" maxlength="20" placeholder="johdoe@email.com" required>
+                <br>
+                <br>
+                <label for="comment">Comment :</label>
+                <textarea name="comment" id="comment" cols="80" rows="10" maxlength="250" placeholder="Type your comment here…" required></textarea>
+                <br>
+                <h4>What is your overall opinion about this article</h4>
+                <input type="radio" id="positive" name="feedback" value="positive">
+                <label for="positive">Positive</label><br>
+                <input type="radio" id="negative" name="feedback" value="negative">
+                <label for="negative">Negative</label><br>
+                <input type="radio" id="neutral" name="feedback" value="neutral">
+                <label for="neutral">Neutral</label>
+                <br>
+                <br>
+                <input type="checkbox" id="subscribe" name="subscribe" value="subscribe">
+                <label for="subscribe">I want to subscribe to the newsletter</label> 
+                <button type="submit">Comment</button>
```

Comment:
Instead of writing inside the button tags, add the value attribute to it and don't forget to change the "Comment" to "Send"

Reply:
Already done!